deployers/servicedeployer.go

```diff
@@ -877,51 +906,54 @@ func (deployer *ServiceDeployer) deleteTrigger(trigger *whisk.Trigger) error {
 }
 
 func (deployer *ServiceDeployer) deleteFeedAction(trigger *whisk.Trigger, feedName string) error {
-
+	deployer.mt.Lock()
+	defer deployer.mt.Unlock()
 	params := make(whisk.KeyValueArr, 0)
 	params = append(params, whisk.KeyValue{Key: "authKey", Value: deployer.ClientConfig.AuthToken})
 	params = append(params, whisk.KeyValue{Key: "lifecycleEvent", Value: "DELETE"})
 	params = append(params, whisk.KeyValue{Key: "triggerName", Value: "/" + deployer.Client.Namespace + "/" + trigger.Name})
 
-    parameters := make(map[string]interface{})
```

Comment:
@pritidesai what editor is doing space conversions?  Do we need to add a formatter in the future?

Reply:
I am running `go fmt <go file>`